### ISSUE TITLE
Feature Request: Enable right click?  & Reply all.

### ISSUE THREAD

**Issue by nportelli** [cezarsa/silver_bird#316]
I'm a terrible speller and NEED Chrome's built in spell checker.  Unfortunately Silver bird doesn't let me right click on the word and correct it.  I'm not sure if this possible with extensions or just disabled. 

Also a replay all feature would be great when multiple names are in a tweet.

**Comment by xingped** [comment]
Agreed and Agreed. The reply all feature is a huge must. I get really irritated having to re-type everyone's name that I'm replying to in the conversation I'm having.

**Comment by rsmarshall** [comment]
Another vote for "reply to all", it's a major key feature missing from silver bird.

**Comment by ChaseOnTheWeb** [comment]
What about the checkbox on the Options screen labeled "Reply to all mentioned"?

**Comment by rsmarshall** [comment]
That I had not seen. :)

**Comment by rsmarshall** [comment]
Also regarding the original poster, I can right click in silver bird and choose spellings.
